example_site/src/components/TemplateBrowser.js

```diff
@@ -73,7 +88,7 @@ function Filter({
         onClick={() => {
           onSelectFilter(filter)
         }}>
-        <div style={{'color': filterNameColor}}>
+        <div style={{color: filterNameColor, 'font-weight': filterFontWeight}}>
```

Comment:
Let's make this a little easier to read and format like this:
```
<div
    style={{
        color: filterNameColor,
        fontWeight: filterFontWeight,
    }}
/>
```

Also, you can convert CSS properties that are hyphenated to camelcase when you use them in JavaScript